ansible/roles/postgresql/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+# PostgreSQL-related settings
+postgresql_is_local: "{{ 'true' if project_db_host | search( 'localhost|127\\.0\\.0\\.1|^/') else 'false' }}"
```

Comment:
Took me a minute to see ^/ here. We should probably add the IPv6 loopback ::1 too.